frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.styled.tsx

```diff
@@ -23,9 +22,7 @@ export const DataSelectorTablePickerHeaderClickable = styled.span<Props>`
   align-items: center;
   display: flex;
 
-  ${({ onClick }) => css`
-    cursor: pointer;
-  `}
+  cursor: pointer;
```

Comment:
What's this change?

Reply:
I guess we used to use `onClick` in the styled component, but not anymore.  This change just remove the use of `css` util since it's unnecessary.